polaris-react/src/components/Inline/Inline.tsx

```diff
@@ -12,6 +15,8 @@ type Align =
   | 'space-evenly';
 type BlockAlign = 'start' | 'center' | 'end' | 'baseline' | 'stretch';
 
+type Spacing = ResponsiveProp<SpacingSpaceScale>;
```

Comment:
Nit, can we rename this type to `Gap` to align with the prop name? I know we want to keep the connection to the spacing tokens and I think the type value including `SpacingSpaceScale` is indicative enough.

```suggestion
type Gap = ResponsiveProp<SpacingSpaceScale>;
```